wordpress-mobile/WordPress-iOS
# Dynamic Cards Analytics

Fixes #22279

This PR adds Dynamic Cards analytics tracking.
- dynamic_dashboard_card_shown
- dynamic_dashboard_card_tapped
- dynamic_dashboard_card_cta_tapped
- ~dynamic_dashboard_card_hide_tapped~ my_site_dashboard_card_hide_tapped ( existing event ).

##  Test Instructions

1. Install Jetpack and login.
2. Enable `Dynamic Cards` feature flag.
3. Navigate back to `My Site`.
4. **Expect** to see **New: Domain Management** dynamic card.
5. **Expect** the event `dynamic_dashboard_card_shown <id: domain_management_announcement>` to be tracked.
6. Tap the card.
7. **Expect** the event `dynamic_dashboard_card_tapped <id: domain_management_announcement, url: https://wordpress.com/domains/manage>` to be tracked.
8. Tap "Check it out" CTA.
9.  **Expect** the event `dynamic_dashboard_card_tapped <id: domain_management_announcement, url: https://wordpress.com/domains/manage>` to be tracked.
10. Tap 3-dots button.
11. **Expect** the event `my_site_dashboard_contextual_menu_accessed <card: dynamic, id: domain_management_announcement>` to be tracked.
12. **Expect** the event `my_site_dashboard_card_hide_tapped <card: dynamic, id: domain_management_announcement>` to be tracked.

## Regression Notes
1. Potential unintended areas of impact
None.

13. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

14. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
